packages/common/amplify_db_common/android/.gitignore

```diff
@@ -0,0 +1,9 @@
+*.iml
+.gradle
```

Comment:
Why are we adding android, ios, linux, macos, windows folders outside of the example sub directory?

Reply:
I think they were autogenerated but have no use in our context here right?  If so can we remove it to reduce the # of files being added in the PR.